### ISSUE TITLE
When importing sites from a Bundle it is possible to have two default sites.

### ISSUE THREAD

**Issue by fabrizzio-dotCMS** [dotCMS/core#22127]
**Describe the bug**
It is possible to have multiple sites marked as _default_ if they get imported from a bundle
If you run the graphQL collection of curl test you'll see that there are two sites marked as default.
this has implications that might affect the logic of the application. 

For example.  After running the GraphQL collection of curl Tests I can not create a content-type imported from a bundle that was generated from an empty starter (Which also comes with a default site). An exception arises importing the new Content-Type with the following stack trace

```
8a7d5e23-da1e-420a-b4f0-471e7da8ea2d': 'Contentlet with id:`48190c8c-42c4-46af-8d1a-0cd5db894797` and title:`test.dotcms.com
test2.dotcms.com
localhost
127.0.0.1` has invalid / missing field(s).
List of non valid fields
PATTERN: description/Description,
```
The problem comes from a call to HostAPI.updateDefaultHost(..) that is originated from ConetenHandler when importing a Bundle

![Screen Shot 2022-05-02 at 13 02 10](https://user-images.githubusercontent.com/37148755/166308836-102bf96f-ca3c-467a-b155-d8ba395c59af.png)

As you can see here there is more than one default site. Which constitutes an invalid state. There can only be one default site. 

![Screen Shot 2022-05-02 at 11 32 13](https://user-images.githubusercontent.com/37148755/166307539-e898ead1-41f5-49ec-831c-193d7735e6e0.png)

**To Reproduce**
1. Setup a fresh instance of dotCMS using the empty starter.
2. Run the first pre_Import_Bundle API calls from our GraphQL Tests
![Screen Shot 2022-05-02 at 13 09 11](https://user-images.githubusercontent.com/37148755/166309713-92a33ae3-fd1e-447b-9a57-18895fe55743.png)

3. Verify on the UI that there is more than one site marked default 
4. Now import the following bundle 
[issue-21482-01G1VQ6TRFC2JQXAM8T10XER4A.tar.gz](https://github.com/dotCMS/core/files/8605514/issue-21482-01G1VQ6TRFC2JQXAM8T10XER4A.tar.gz)
5. See how the import fails with a similar error in the logs.

**Expected behavior**
We should be able to import the bundle without having a conflict updating the current default host.

**Comment by fabrizzio-dotCMS** [comment]
Related with https://github.com/dotCMS/core/pull/22109

**Comment by fabrizzio-dotCMS** [comment]
related to https://github.com/dotCMS/core/issues/21482

**Comment by fabrizzio-dotCMS** [comment]
This is **NOT** a real issue. When I imported the bundle that I was using to test https://github.com/dotCMS/core/issues/21482
While debugging the error validating the site that is reported here. I mistakenly thought we had a problem with several default sites at the same time.
But this situation was fixed by the code changes pushed into the module enterprise  in the following PR https://github.com/dotCMS/enterprise/pull/872

This fixed the exception validating the imported site allowing the code in the method **HostAPIImpl.updateDefaultHost** to work properly
